sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs

```diff
@@ -36,35 +35,58 @@ namespace Azure.Storage.Test.Shared
         LiveServiceVersions = new object[] { StorageVersionExtensions.LatestVersion })]
     public abstract class BlobTestBase : StorageTestBase<BlobTestEnvironment>
     {
+        /// <summary>
+        /// Source of clients.
+        /// </summary>
+        protected ClientBuilder<BlobServiceClient, BlobClientOptions> Clients { get; }
```

Comment:
nit: By looking at name I'd guess that it's a `List<Client>`. I'd be verbose in property naming, with IDE support that shouldn't be a big deal.